components/unzipped/dashboard/DashboardTable.js

```diff
@@ -86,46 +87,63 @@ const Panel = ({list, selectedList, type, projects=[], businesses, loading}) =>
                 <DarkText noMargin center bold>Actions</DarkText>
             </TableTitle>
-            <Underline color="#333" noMargin/>   
+            <Underline color="#333" noMargin />
             <StoryTable>
                 {businesses.length === 0 && loading && <Box><CircularProgress /></Box>}
                 {businesses.length === 0 && <Box>Start a project and you will see it here...</Box>}
                 {businesses.length > 0 && businesses.map((item, index) => (
                     <WhiteCard noMargin borderRadius="0px" row background={!ValidationUtils.checkNumberEven(index) ? "#F7F7F7" : "#fff"}>
-                            <Absolute width="45%" wideLeft textOverflow="ellipsis"><DarkText textOverflow="ellipsis" noMargin>{item.name}</DarkText></Absolute>
-                            <DarkText noMargin> </DarkText>
-                            <DarkText noMargin> </DarkText>
+                        <Absolute width="45%" wideLeft textOverflow="ellipsis"><DarkText textOverflow="ellipsis" noMargin>{item.name}</DarkText></Absolute>
+                        <DarkText noMargin> </DarkText>
+                        <DarkText noMargin> </DarkText>
                         <DarkText noMargin center>${(item.budget || 0).toLocaleString()}</DarkText>
                         <DarkText noMargin center>{item?.equity || 0}%</DarkText>
                         <DarkText noMargin center>27</DarkText>
                         <DarkText noMargin center>{item?.valueEstimate || 'N/A'}</DarkText>
                         <DarkText noMargin center>{(item?.deadline && ValidationUtils.formatDate(item?.deadline)) || ValidationUtils.formatDate(item?.updatedAt || item?.createdAt)}</DarkText>
                         <DarkText noMargin center></DarkText>
                         <Absolute>
-                        <Button
-                            icon="largeExpand"
-                            popoutWidth="150px"
-                            noBorder
-                            block
-                            type="lightgrey"
-                            fontSize="13px"
-                            popout={[
-                                {
-                                    text: 'Details',
-                                    onClick: () => console.log('ITEM 1'),
-                                },
-                                {
-                                    text: 'Delete Job',
-                                    onClick: () => console.log('ITEM 2'),
-                                },
-                                {
-                                    text: 'Assign department',
-                                    onClick: () => console.log('ITEM 3'),
-                                },
-                            ]}
-                            iconRight>
-                            Details
-                        </Button>
+                            <Button
```

Comment:
This logic can be moved to a separate function. Please try to keep UI components and business logic separate. Thanks